PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -31,13 +31,13 @@
 
 void printInt(int int2print)
 {
-	TCHAR str[32];
+	wchar_t str[32];
 	wsprintf(str, TEXT("%d"), int2print);
 	::MessageBox(NULL, str, TEXT(""), MB_OK);

```

Comment:
now it should be:
```
	wsprintfW(str, L"%d", int2print);
	::MessageBoxW(NULL, str, L"", MB_OK);
```

Reply:
@xomx 
Agree for the TEXT("") and _T('') being replaced by the L"", to make the intention (semantic) of using char or wchar_t more clear. However, adding suffix "W" for all the win32 functions is not necessary to me.